### ISSUE TITLE
"Save As" function adding extra carriage return to all lines

### ISSUE THREAD

**Issue by xarxziux** [lucidlylogicole/scope#1]
Hi,

I've been trying out Scope for only a few days now and have come across a curious problem with the "Save As" function.  It appears to be adding an extra carriage return after every line, which is giving the appearance of an extra blank line after each original line.

For example, see the two text files below.  The first one, when viewed with an editor that can show all characters looks like this:

	Line 1<CR><LF>
	Line 2<CR><LF>
	Line 3<CR><LF>
	

However, when this is saved with the "Save As" function it becomes this:

	Line 1<CR>
	<CR><LF>
	Line 2<CR>
	<CR><LF>
	Line 3<CR>
	<CR><LF>
	

I've noticed the program doing this with .js, .vbs and .txt files, so it seems to be the IDE that's doing this, and not related to a particular editor.

This occurred under Scope version 0.6.17, running on Windows 7 Professional.  I haven't yet checked this on other computers or OS'es.

[scope test.txt](https://github.com/lucidlylogicole/scope/files/175765/scope.test.txt)
[scope test1.txt](https://github.com/lucidlylogicole/scope/files/175766/scope.test1.txt)

**Comment by lucidlylogicole** [comment]
Do you know which editor is being used - Scintilla or Ace?  Or are you just using the default settings?

**Comment by xarxziux** [comment]
Apologies for the delay in getting back to you.  I've been on holidays for a while, so I didn't see your reply earlier.

I did some tests just now and it appears to be independent of the editor.  I tried it with a text file, a Python file and JavaScript file, and all of them do the same thing.  I'm not sure how to verify what editor is being used but it should just be the default settings, so Scintilla for the text and python files and Ace for the JavaScript files.

I've also discovered that using Save As on a file that already has an extra CR added will add yet another CR and so on.

I've yet to test this problem on my home computers.  I'll let you know if I get the chance to do so.
